client/js/views/GameView.vue

```diff
@@ -103,15 +103,14 @@
                     name="slide-below"
                     class="opponent-hand-wrapper transition-all"
                   >
-                    <div
+                    <card
                       v-for="(card, index) in opponent.hand"
-                      :key="index + 0"
+                      :key="index"
+                      suit="back"
+                      rank="back"
+                      data-opponent-hand-card
                       class="transition-all opponent-card-back-wrapper opponent-hand-card mx-2"
-                    >
-                      <v-card class="opponent-card-back" data-opponent-hand-card>
-                        <v-img :src="require('../img/logo_head.svg')" contain />
-                      </v-card>
-                    </div>
+                    />
```

Comment:
With the removal of required for the `rank` and `suit` props, this could be simplified to just 
```
                    <card
                      v-for="(card, index) in opponent.hand"
                      :key="index"
                      data-opponent-hand-card
                      class="transition-all opponent-card-back-wrapper opponent-hand-card mx-2"
                    />
```